docs/README.md

```diff
@@ -0,0 +1,86 @@
+# Determined docs development
+
+Determined uses Sphinx to build docs, but with some functional enhancements. 
+This is a partial list of those enhancements.
```

Comment:
To simplify:

This document lists some of the enhancements Determined has made to its Sphinx-generated documentation.

(Instead of:
Determined uses Sphinx to build docs, but with some functional enhancements. 
This is a partial list of those enhancements.)